doc/userDocs/usersGuide.md

```diff
@@ -152,11 +152,18 @@ The image below shows the lines that need to be edited in the configuration file
           stack of your cluster. The cluster on which the applications run must 
           support provisioning external IP addresses for LoadBalancer services. 
     - `nodeport`: Traffic from the workload drivers to the applications will
-          use the external IP addresses of the Kubernetes Nodes via a NodePort service. This 
+          use the *external* IP addresses of the Kubernetes Nodes via a NodePort service. This 
           may be desirable even when the workload drivers and applications are running 
           on the same cluster in order to load-test the complete networking 
           stack of your cluster. The cluster on which the applications run must 
           support exposing the node IP addresses externally using NodePort services.
+    - `nodeport-internal`: Traffic from the workload drivers to the applications will
+          use the *internal* IP addresses of the Kubernetes Nodes via a NodePort service. This 
+          may be desirable even when the workload drivers and applications are running 
```

Comment:
is the word 'even' needed here?

Reply:
I think so, since the main use of either nodeport or nodeport-internal will be when the drivers and applications are in different clusters, but you might want to use them even when they are in the same cluster.